requirements-dev.txt

```diff
@@ -7,7 +7,7 @@ pytest-cov
 pytest-xdist
 setuptools
 wheel
-ml_dtypes
+ml_dtypes>=0.5.0
```

Comment:
```suggestion
```

I think we can remove instead, because this is specified as a package dependency already